src/charge_controller/ChargeController.cpp

```diff
@@ -68,18 +87,20 @@ void ChargeController::noBatteryState() {
  * if bms is ready to charge, wait in standby for user to start charge
  */
 void ChargeController::connectedState() {
-  if (changedState) {
-    // display connected LCD message
-    // further init battery communication?
-    changedState = false;
-  }
-  if (checkBMS()) {
-    changedState = true;
-    state = ControllerStates::STANDBY;
-  } else {
-    changedState = true;
-    state = ControllerStates::FAULT;
-  }
+    if (changedState) {
+        LOG.log(Logger::DEBUG, "Changed state->Connected State");
+        LOG.log(Logger::DEBUG, "%d pack(s) connected", bms.numConnected());
+        // display connected LCD message
+        // further init battery communication?
+        changedState = false;
+    }
+    if (checkBMS() == 0) {
+        changedState = true;
+        state = ControllerStates::STANDBY;
+    } else {
+        changedState = true;
```

Comment:
changedState will always end up being true and it does not look like the checkBMS method uses this field. Is it intentional or can it be simplified?